src/lib/modules/guilds/types.ts

```diff
@@ -0,0 +1,29 @@
+import type { RecordModel } from "pocketbase";
+import type { User } from "../authentication";
+
+
+
+export interface Guild extends RecordModel {
+    name : string,
+    descriptions : string,
+    owner_id : User,
```

Comment:
```suggestion
    owner_id : string,
```
Hello @arielrecto , the `owner_id` is a `string` type based from the db, if you want to properly type a model relationship you should use

```suggestion
    expand? : { owner_id: User },
```